gui/Test/src/main/java/sample/test/controllers/ManagerViewController.java

```diff
@@ -0,0 +1,214 @@
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.control.Button;
+import javafx.stage.Stage;
+import sample.test.Main;
+import sample.test.model.User;
+import sample.test.service.JwtTokenService;
+import sample.test.service.UserService;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.URL;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.List;
+import java.util.ResourceBundle;
+
+public class ManagerViewController implements Initializable {
+
+    @FXML
+    private AnchorPane dashboardPane;
+    @FXML
+    private AnchorPane menuPane;
+    @FXML
+    private AnchorPane staffPane;
+    @FXML
+    private AnchorPane ordersPane;
+    @FXML
+    private Button closeButton;
+    @FXML
+    private Button signOutButton;
+    @FXML
+    private Button menuButton;
+    @FXML
+    private Button staffButton;
+    @FXML
+    private Button ordersButton;
+    @FXML
+    private Button userAddButton;
+    @FXML
+    private Button userEditButton;
+    @FXML
+    private Label userNameLabel;
+    @FXML
+    private TableView<User> userTableView;
+    @FXML
+    private TableColumn<User, Integer> idColumn;
+    @FXML
+    private TableColumn<User, String> usernameColumn;
+    @FXML
+    private TableColumn<User, String> rolesColumn;
+
+    public void initialize(URL url, ResourceBundle rb) {
+        userNameLabel.setText(UserService.getInstance().getUsername());
+        showDashboardPane();
+        initUserTable();
+    }
+
+    private void initUserTable() {
+        idColumn.setCellValueFactory(new PropertyValueFactory<>("id"));
+        usernameColumn.setCellValueFactory(new PropertyValueFactory<>("username"));
+        rolesColumn.setCellValueFactory(new PropertyValueFactory<>("roles"));
+        loadUserData();
+    }
+
+    private void loadUserData() {
+        try {
+            String jsonResponse = makeHttpRequest();
+            List<User> users = parseJsonResponse(jsonResponse);
+            populateTableView(users);
+
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    private String makeHttpRequest() throws Exception {
+        HttpClient client = HttpClient.newHttpClient();
+        String token = JwtTokenService.getInstance().getJwtToken();
+        String url = "http://localhost:8080/users/";
+
+        HttpRequest request = HttpRequest.newBuilder()
```

Comment:
you will propably use this logic a lot, extract this method and make it universal